content/pages/roadmap.html

```diff
@@ -16,61 +16,98 @@ <h4><i class="fa fa-map"></i> <b>Roadmap</b></h2>
 
-                <h1><i class="green_font fa fa-angle-right"></i> Upcoming Release: iRODS 4.3</h1>
+                <h1><i class="green_font fa fa-angle-right"></i> Upcoming Release: iRODS 4.4</h1>
 
-                <p>The focus for iRODS 4.3 will be on two things: the logging framework, and hardening the existing features and new work introduced in 4.2.</p>
+                <p>The primary focus of 4.4 is simplification and maintainability.</p>
 
-                <p>Alongside this work, but not part of the core codebase, the Consortium will be working on supporting features and functionality provided by additional external tools and features delivered as plugins.</p>
+                <p>Making changes to iRODS is difficult. With the release of the 4.3 series, the Consortium strived to improve the implementation by cleaning up semantics. This was accomplished through various techniques. For example, by introducing higher level abstractions, the implementation became easier to reason about while also allowing the internals to be changed without fear of breaking existing deployments. iRODS 4.4 will continue this effort by attempting to address the issues outlined below.</p>
 
-                <h3>Logging</h3>
+                <h2>The iRODS Protocol</h2>
 
-                <p>The logging in iRODS needs to be modernized and standardized with regards to best practices around log rotation, timestamps, and human and machine readable formats.  We plan to investigate a variety of logging frameworks and select one that delivers flexibility powerful enough to afford easy aggregation of logs from our distributed system.</p>
+                <p>iRODS implements its own protocol. It is not documented or versioned. Attempting to add support for other programming languages is a real challenge to developers unfamiliar with iRODS.</p>
 
-                <h3>Hardening</h3>
+                <div>
+                    <p>To make iRODS more accessible, the Consortium will be investigating the following approaches:</p>
 
-                <p>The everpresent backlog of bugfixes and edge cases will continue to be addressed.  This release will mark the first of a new tick-tock release pattern, meaning one release will focus on features, and the next will focus more on hardening.  4.2 was the feature release, and 4.3 will be the hardening release.</p>
+                    <h3>Single Standardized Protocol</h3>
+                    <div>
+                        <p>The idea of moving to a standardized protocol was motivated by the efforts behind the C++ REST API.</p>
 
-                <h3>Multipart Data Objects</h3>
+                        <p>The HTTP protocol is the primary candidate here. It is simple to understand and adding support for it is literally a matter of choosing a good library. S3 has seen major success and it is built entirely on the HTTP protocol. So, it makes a lot of sense to choose HTTP for iRODS. Doing that instantly widens the reach of iRODS.</p>
```

Comment:
"it makes a lot of sense" -> "it may make a lot of sense"

Reply:
done